keycloak/user.go

```diff
@@ -0,0 +1,114 @@
+package keycloak
+
+import "github.com/Nerzal/gocloak/v11"
+
+const (
+	// KeycloakDefaultOrganizationRef references the keycloak user attribute.
+	// TODO(bastjan) If we add more attributes I'd use struct tags struct{ DefaultOrganizationRef string `kcattr:"appuio.io/default-organization"` }
```

Comment:
Wouldn't call this a TODO, but I definitely agree

Reply:
If we think using struct tags would be a good idea, what's the rationale for not just doing it now?

Reply:
My rationale for not directly using struct tags would be YAGNI. Custom struct tags are IMO a bit finicky and overkill for a single tag.

Reply:
It's quite some work if you wanna do it the right way™️ and have some good tests too. Maybe if i got time at the end of the sprint...